starter-code/functions-and-arrays.js

```diff
@@ -1,14 +1,34 @@
 // Find the maximum
 function maxOfTwoNumbers (first, second) {
-
+  if (first>second){
+    console.log ("first");
+  }
+  else if (second>first){
+    console.log ("second");
+  }
+  else {
+    console.log("the numbers are the same");
+  }
 }
 
 var largest = maxOfTwoNumbers(2, 6);
 console.log(largest);
 
 // Finding Longest Word
-function findLongestWord (words) {
 
+function findLongestWord (words) {
+    var biggest="";
+    for ( var ix=0; ix < words.length; ix++){
+      if ( biggest.length < words[ix].length){
+        biggest=words[ix];
+      }
+    }
+    for ( var j=0; j<words.lenght; j++){
```

Comment:
You don't need this second for. I understand you use it to display more than one result (if there is a tie for the longest word), but the problem again is that you MUST return the result instead of showing it in the console. In this case, if you want to display more than one word, you will need an array. If the function's signature is expecting an output of a value, returning an array could be a problem.